pkg/probe/exec/exec_test.go

```diff
@@ -24,14 +24,19 @@ import (
 )
 
 type FakeCmd struct {
-	out []byte
-	err error
+	out    []byte
+	stdout []byte
```

Comment:
why not just use the same output field?  We don't expect to  us both Output() and CombinedOutput() in the same test case, do we?